ui/Screen/Project.svelte

```diff
@@ -43,6 +43,10 @@
     selectPeer(peer);
   };
 
+  const isContributor = (userUrn: Urn, users: User[]): boolean => {
```

Comment:
You can express that without any extra information. In the user array if there is one element which is of `peerType` local and has either one of two roles: `Contributor`, `Maintainer` - we know the project has been "forked" or started by the local user.

While your approach is likely to work with how the current peer selection is assembled this can break subtly, as the heuristics that filters out a local peer which only tracks might be changed. Additionally to keep components - ideally -  concerned with presentation; it would be great to move a function like this one close to the types (`project.ts`), to then be imported here.